src/main/java/com/bbmk/restaurant/service/UserService.java

```diff
@@ -65,4 +65,47 @@ public void delete(User user) {
 		System.out.println("Delete Successful");
 	}
 
+	@Override
+	public void activateUser(User user) {
+		user.setActive(true);
+		userRepo.save(user);
+		System.out.println("Activate Successful");
+	}
+
+	@Override
+	public void deactivate(User user) {
+		user.setActive(false);
+		userRepo.save(user);
+		System.out.println("Deactivate Successful");
+
+	}
+
+	@Override
+	public void changePassword(User user) {
+		user.setPassword(passwordConfig.bCryptPasswordEncoder().encode(user.getPassword()));
+		userRepo.save(user);
+		System.out.println("Change Password Successful");
+
+	}
+
+	@Override
+	public void changeUsername(User user) {
+		user.setUsername(user.getUsername());
+		userRepo.save(user);
+		System.out.println("Change Username Successful");
+
+	}
+
+
```

Comment:
Always leave 1 line after each method/function

Reply:
Go to UserController